docs/overview/about.md

```diff
@@ -68,12 +79,16 @@ These Archway-specific modules extend the functionality of the Cosmos SDK.
 
 To achieve the Archway architecture [desired properties](#the-archway-mission), the gas tracking module follows this design:
 
-- Wrap the [CosmWasm](./glossary.md#cosmwasm) VM with an Archway GasMeter to allow interception and tracking of information passed by the VM.
-- During BeginBlock, the recorded information is brought to memory and processed to determine rewards for each of the contracts executed in the last block. 
-- Developers can choose to take the rewards or give the gas rebate to end users to subsidize transaction costs. To enable that choice, use a proxy wrapper. The wrapper expects the developer to specify configuration parameters during the instantiate call. The parameters are related to the gas reward and the instantiation request body. This body is consumed by the wrapper and the contract is sent only instantiation requests which means the contract interface does not need to be changed.
-- Developer reward payout can happen on demand or automatically at BeginBlock. 
-    - The disadvantage of on-demand reward is an inferior user experience and results in more transactions just to transfer the rewards. 
-    - The advantage of on-demand reward is that the reward is automatically transferred to the designated reward address when it crosses a certain threshold.  
+1) Wrap the [CosmWasm](./glossary.md#cosmwasm) VM with an Archway GasMeter to allow interception and tracking of information passed by the VM.
+  
+2) During BeginBlock, the recorded information is brought to memory and processed to determine rewards for each of the contracts executed in the last block. 
+
+3) Developers can choose to take the rewards or give the gas rebate to end users to subsidize transaction costs. To enable that choice, use a proxy wrapper. 
+    The wrapper expects the developer to specify configuration parameters during the instantiate call. The parameters are related to the gas reward and the instantiation request body. This body is consumed by the wrapper and the contract is sent only instantiation requests which means the contract interface does not need to be changed.
+
+4)  Developer reward payout can happen on demand or automatically at BeginBlock. 
+    The disadvantage of on-demand reward is an inferior user experience and results in more transactions just to transfer the rewards. 
+    The advantage of on-demand reward is that the reward is automatically transferred to the designated reward address when it crosses a certain threshold.  
```

Comment:
The information on `3` and `4` is outdated.

Reply:
hey @aelesbao where can we find the updated info regarding points `3` and `4`?